### DISCUSSION TITLE
Set-Cookie header is ignored

### DISCUSSION THREAD

**Discussion by aboettger-tuhh** [qutebrowser/qutebrowser#7379] [Q&A] [answered]
**Version info**:
https://paste.the-compiler.org/view/fc2865aa

**Description**
Set-Cookie header is ignored in response from url: https://[snip]. Cookie length should be less than or equal to 4096 characters.

Can I bypass the restriction?

**Comment by The-Compiler** [comment, marked as answer]
Please use discussions for questions instead of opening a bug report. I converted it to one.

The limit is [coming from the underlying Chromium](https://source.chromium.org/chromium/chromium/src/+/main:net/cookies/parsed_cookie.h;l=29;drc=ef611f3beff75b4c745f69abd7d57feeaaf35d15) and doesn't seem to be configurable.
